### DISCUSSION TITLE
hyprswitch is just working

### DISCUSSION THREAD

**Discussion by aloispichler** [H3rmt/hyprshell#84] [Show and tell]
Just wanted to report that I used hyprswitch on two monitors the first time today. Everything’s working out of the box, looks great, very convincing. No crashes since months on my other machine. Wonderful app!

**Comment by H3rmt** [comment]
You can try to remove or add monitor(s) while the daemon has already started, that will definetly confuse hyprswitch (I am working on a fix but currently struggle with async rust + gtk)
